src/db/models/index.ts

```diff
@@ -1,14 +1,21 @@
 'use strict';
-const path = require('path');
-const Sequelize = require('sequelize');
-const config = require('config');
-const logger = require('../../core/log');
+
+import path from 'path';
+import {Sequelize} from 'sequelize';
+import config from 'config';
+import logger from '../../core/log';
+import {resolveHome} from '../../core/utils';
+
 const log = logger.child({module: 'Sequelize'});
-const {resolveHome} = require('../../core/utils');
 
-class SequelizeFactory {
-    init(ctx) {
-        this.ctx = ctx;
+interface SequelizeFactory {
+    config: DatabaseConfig;
+    Sequelize: typeof Sequelize;
```

Comment:
maybe we should install types for Sequelize

Reply:
Already did [here](https://github.com/pointnetwork/pointnetwork/pull/236/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R76).